product_variant_sale_price/__manifest__.py

```diff
@@ -5,10 +5,10 @@
 {
     "name": "Product Variant Sale Price",
     "summary": "Allows to write fixed prices in product variants",
-    "version": "17.0.1.1.0",
+    "version": "17.0.1.1.1",
```

Comment:
You don't need to manually increment the version; the bot will do this during merge.